mmf/models/mmf_transformer.py

```diff
@@ -287,31 +287,35 @@ def preprocess_sample(self, sample_list: Dict[str, Any]) -> BaseTransformerInput
 
         return BaseTransformerInput(input_ids, position_ids, segment_ids, masks)
 
-    def forward(self, sample_list: Dict[str, Any]) -> Dict[str, Tensor]:
+    def transformer_encoder(self, embedding, attention_mask):
```

Comment:
The function names looks like a property. We should name it maybe `transfomer_encode` to avoid confusion.